test/suites/optional/generic-device-plugin.robot

```diff
@@ -49,6 +49,29 @@ Verify ttyUSB glob pattern device discovery and allocation
 
     [Teardown]    TtyUSB Glob Test Teardown
 
+Verify FUSE device allocation and accessibility
+    [Documentation]    Verifies FUSE device configuration, allocation, and accessibility in pods
+    [Tags]    fuse-device
+    Enable And Configure GDP    ${GDP_CONFIG_FUSE_COUNT}
+    Wait Until Device Is Allocatable    10    fuse
+    # Create and deploy pod requesting 4 FUSE devices
+    ${path}=    Create Random Temp File    ${POD_FUSE_DEVICE}
```

Comment:
How about moving the string into a dedicated file in test/assets/generic-device-plugin/?

Reply:
okay, i can do that.